railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -68,7 +68,7 @@ def gitignore
 
     def version_control
       if !options[:skip_git] && !options[:pretend]
-        run "git init", capture: options[:quiet]
+        run "git init", capture: true
```

Comment:
`capture: true` will suppress the output of the command even when `--quiet` wasn't passed, which isn't desirable. The fact that it causes ENOENT to be raised when the command is invalid is an undocumented side effect.

Instead I think we should pass `abort_on_failure: false`, which will restore the previous behaviour and avoid making the user pass `--skip-git` when git isn't available:

```suggestion
        run "git init", capture: options[:quiet], abort_on_failure: false
```

Reply:
Ah yes, that seems a lot better. Thanks for the feedback. I will test and revise the PR.